### PR TITLE
Feature/user interface

### PR DESCRIPTION
**Pull request by GuNner-pete**
Assigning players' names and telling players to make their next moves

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env ruby
+class Player
+  attr_accessor :name, :movement
+
+  def initialize(name, movement)
```
Comment (GzimAsani) [inline bin/main.rb:5]: _Kindly fix this_
bin/main.rb:5:in `initialize': wrong number of arguments (given 0, expected 2) (ArgumentError)

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED  :stop_sign: 
### HELLO **TEAM** You did a great job so far  :+1: but unfortunately your code breaks on line 5:
- [x]  The code breaks in  bin/main.rb:5:in `initialize': wrong number of arguments (given 0, expected 2) (ArgumentError), kindly fix this
### Also:
- [x] game tells which player turn it is (without computing who the winner is)
- [x] game informs player if the selected move is invalid (without computing what an invalid move is)
- [x] game informs player if the selected move is a winning move (without computing the winning move)
- [x] game informs player if the selected move is a draw move (without computing draw)
- [x] game repeats all actions for the next player's move
- [x] A loop is needed in this game (recommended is `while` but anything else will do) ([OdinProject hint](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/loop.png))

### **Please make the required changes and submit another review request when you are ready.**

<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:

File: bin/main.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  puts 'X will be player one, what is your name?'
+  player1.name = gets.chomp
+  puts "Hello #{player1.name}, you are now the X"
+  puts 'O will be player two, what is your name? (Please insert a nickname)'
+  player2.name = gets.chomp
+  puts "Hello #{player2.name}, you are now the O"
+  puts ' --------------------------------------------------------------- '
+  puts
+  puts
+  puts "Ready to play #{player1.name} and #{player2.name}? Let's go!"
```
Comment (BrianSammit) [inline bin/main.rb:33]: I can't see these puts on the terminal, it just skips to the game. 

CavernousEnergeticBluetonguelizard-size_restricted

**Review by BrianSammit — APPROVED** [review]
STATUS: Approve :muscle: 
HELLO TEAM!! :wave:  Excellent job working on the changes, the project is working amazing :smiley: 
---------------------------------------------------------------

- [x] [OPTIONAL] fix the last two lines when the game starts. See my comments to see where is the bug.

--------------------------------------------------------

You can merge this milestone :clap: 

HAPPY CODING!!! Good luck with the next milestones. 

MB5W4D

**Pull request merged**